src/internal_events/add_fields.rs

```diff
@@ -57,7 +57,7 @@ pub struct AddFieldsFieldOverwritten<'a> {
 
 impl<'a> InternalEvent for AddFieldsFieldOverwritten<'a> {
     fn emit_logs(&self) {
-        error!(message = "Field overwritten.", %self.field, rate_limit_secs = 30);
+        debug!(message = "Field overwritten.", %self.field, rate_limit_secs = 30);
```

Comment:
We need to consider 2 possibilities here, each of which is of different importance.

* The field exists, and should have been overwritten. This should be `debug`. (A user would appreciate this information while debugging)
* The field does not exist and should have been overwritten. This should be `trace` (we did not need to take action, but tracing that we did check if we needed to take action is good)